scripts/build_demand_profiles.py

```diff
@@ -0,0 +1,209 @@
+# -*- coding: utf-8 -*-
+# SPDX-FileCopyrightText: : 2017-2020 The PyPSA-Eur Authors, 2021 PyPSA-Africa Authors
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+# coding: utf-8
+"""
+Creates an electric demand file.
```

Comment:
Comment on docstring by max applies here.
Maybe good to add more details for example on what input (e.g. gegis) and output data.

Reply:
Moreover, this file needs to be added to the API documentation.
Need to modify file doc/api_reference.rst and add some lines alike the other scripts/rules

Reply:
I modified the api_reference. I also improved the docstring in build_demand_profiles, hope it's good now.